selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/AmendCommitTest.java

```diff
@@ -88,6 +88,9 @@ public void prepare() throws Exception {
 
   @Test
   public void checkAmendPreviousCommit() {
+    String pathToFile = PROJECT_NAME + "/src/main/java/org.eclipse.qa.examples";
```

Comment:
We have already have `PATH_TO_FILE` constant which looks similar to `pathToFile`. I would name variable differently to avoid confusion.

Reply:
What about the name `pathToFileItem`?